core/primitives/src/version.rs

```diff
@@ -136,12 +138,24 @@ pub enum ProtocolFeature {
 /// Current latest stable version of the protocol.
 /// Some features (e. g. FixStorageUsage) require that there is at least one epoch with exactly
 /// the corresponding version
-#[cfg(not(feature = "nightly_protocol"))]
+#[cfg(all(
+    not(feature = "nightly_protocol"),
+    any(
+        not(feature = "protocol_feature_new_validator_selection_algorithm"),
+        feature = "protocol_feature_chunk_only_producers"
+    )
+))]
 pub const PROTOCOL_VERSION: ProtocolVersion = 49;
 
 /// Current latest nightly version of the protocol.
 #[cfg(feature = "nightly_protocol")]
-pub const PROTOCOL_VERSION: ProtocolVersion = 124;
+pub const PROTOCOL_VERSION: ProtocolVersion = 126;
+
+#[cfg(all(
```

Comment:
please add a proper comment here -- and mention that this should be removed in January 2022, once new_validator is in mainnet

Reply:
As mentioned in PR description, this would be removed before merge

Reply:
As we are ready for merge I reverted this part